README.md

```diff
@@ -1,13 +1,13 @@
-WebExtensions / Chrome browser add-on for linking a browser to the KeePassRPC KeePass plugin.
+WebExtensions (Mozilla Firefox) / Google Chrome (Chromium) browser add-on for linking browser to KeePassRPC KeePass plugin.
 
 Build requirements:
 
-# node
-# npm
-# the usual hundreds of node modules installed by the above
+* node
+* npm or yarn
+* the usual hundreds of node modules installed by the above
 
 It's all set up for Visual Studio Code but it shouldn't be too hard to work out how to build it using other IDEs.
 
 It's not well tested but you should be able to just clone the repo and `npm install` followed by `gulp watch` and then load the root folder as an unpacked extension in Chrome.
 
-In future we can enhance the build process to automate other things like packaging releases for different browsers but we're a long way from having a releasable version yet anyway.
\ No newline at end of file
+In future we can enhance the build process to automate other things like packaging releases for different browsers but we're a long way from having a releasable version yet anyway.
```

Comment:
Seems to have converted to CRLF line endings which is fine by me, not sure why my Windows VS code text editor created this specific file with LF endings unlike all others it created - maybe it thinks there is something special about Markdown or README files but either way, it looks to be in a good state now so thanks :-)